lib/Mojo/WebSocketProxy/Backend/JSONRPC.pm

```diff
@@ -119,16 +120,15 @@ sub call_rpc {
                     my $tx = $client->tx;
                     $req_storage->{req_url} = $tx->req->url;
                     my $err = $tx->error;
-                    my $cb_processed_totally;
-                    $cb_processed_totally = $rpc_failure_cb->(
+                    $rpc_failure_cb->(
                         $c, $res,
                         $req_storage,
                         {
                             code    => $err->{code},
                             message => $err->{message},
                             type    => 'WrongResponse',
                         }) if $rpc_failure_cb;
-                    return if $cb_processed_totally;
+                    return if $failure_block_response;
```

Comment:
Actually I was thinking of a flag `block_response` that would block all responses, success or failure. What do you think?

Reply:
Sorry I misunderstand your last comment. To be honest it is out of this card. I don't know if it will cause some strage behavior . But I think it is a good idea, and I guess it has no big problem.

Reply:
Let me do it now.